### PR TITLE
armor 0.1.2

### PR DESCRIPTION
**Pull request by vishr**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

Updated description about the project and new binary.

Signed-off-by: Vishal Rana vr@labstack.com

### CODE REVIEW

**Comment by vishr** [comment]
Is there a way to update files in https://homebrew.bintray.com?

File: Formula/armor.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Armor < Formula
-  desc "Simple HTTP server, supports HTTP/2 and auto TLS"
+  desc "Uncomplicated HTTP server, supports HTTP/2 and auto TLS"
   homepage "https://github.com/labstack/armor"
   url "https://github.com/labstack/armor/archive/v0.1.2.tar.gz"
-  sha256 "b9106fd828c34f2dce89d7edee0cb3f60788471ba284cbc0fe381990652d463f"
+  sha256 "ddab863215a58d3355a1fbe204e3fe910490e39d4d0c6392c2a52821f7b1bc93"
```
Comment (MikeMcQuaid) [inline Formula/armor.rb]: This should not change unless the tag changes too. Can you ask upstream why this happened and if they've been hacked?

**Comment by MikeMcQuaid** [comment]
> Is there a way to update files in https://homebrew.bintray.com?

@vishr It'll be done automatically when we merge.

File: Formula/armor.rb (hunk 1 above)
Comment (vishr) [inline Formula/armor.rb]: I own this project (Armor). I didn't want to cut a release for description change and a typo so just updated the binaries, I am aware it's not clean.

Comment (vishr) [inline Formula/armor.rb]: Sure. Closing in favor of https://github.com/Homebrew/homebrew-core/pull/5693

Comment (MikeMcQuaid) [inline Formula/armor.rb]: Thanks, really appreciate it 🙇

**Pull request closed without merging**